custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -17,9 +17,9 @@
 TEST_CASE_TYPE = 'test'
 
 
-def validate_number(string_value):
+def validate_phone_number(string_value):
     if string_value is None or string_value.strip() == '':
-        return None
+        return ''
     else:
         return int(string_value)
```

Comment:
@nickpell if all the numbers are exactly 10 digits without a leading zero (and no other characters), then assuming these are all Indian numbers, all we should have to do is add '91' to the beginning of each in order to put it in international format